src/js/view/captionsrenderer.js

```diff
@@ -213,7 +213,7 @@ const CaptionsRenderer = function (viewModel) {
 
         const fontSize = Math.round(height * _fontScale);
 
-        if (_model.get('renderCaptionsNatively')) {
```

Comment:
So we only call `_setShadowDOMFontSize` on Safari Desktop? Do we even need to?

Reply:
The idea behind doing this was to add styles to ensure native/player rendered captions are sized similarly. If it's not working as intended, we should just get rid of it.